### ISSUE TITLE
RuntimeError: CUDA error: CUBLAS_STATUS_NOT_INITIALIZED when calling `cublasCreate(handle)`

### ISSUE THREAD

**Issue by youxingtian** [triton-inference-server/server#4083]
I use this command
```model-analyzer profile --model-repository /models/ --profile-models pytorch_bert --override-output-model-repository```
the error:
```
2022-03-15 09:16:24.792 INFO[metrics_manager.py:335] Profiling model pytorch_bert_config_4...
2022-03-15 09:16:26.831 INFO[perf_analyzer.py:258] Running perf_analyzer ['perf_analyzer', '-m', 'pytorch_bert_config_4', '-b', '1', '-u', 'localhost:8001', '-i', 'grpc', '--concurrency-range', '1', '--measurement-mode', 'count_windows'] failed with exit status 1 : *** Measurement Settings ***
  Batch size: 1
  Using "count_windows" mode for stabilization
  Minimum number of samples in each window: 50
  Using synchronous calls for inference
  Stabilizing using average latency

Request concurrency: 1
Failed to maintain requested inference load. Worker thread(s) failed to generate concurrent requests.
Thread [0] had error: PyTorch execute failure: The following operation failed in the TorchScript interpreter.
Traceback of TorchScript, serialized code (most recent call last):
  File "code/__torch__/transformers/models/distilbert/modeling_distilbert.py", line 17, in forward
    pre_classifier = self.pre_classifier
    distilbert = self.distilbert
    _0 = (distilbert).forward(input_ids, attention_mask, )
          ~~~~~~~~~~~~~~~~~~~ <--- HERE
    _1 = torch.slice(_0, 0, 0, 9223372036854775807)
    input = torch.select(_1, 1, 0)
  File "code/__torch__/transformers/models/distilbert/modeling_distilbert.py", line 35, in forward
    transformer = self.transformer
    embeddings = self.embeddings
    _3 = (transformer).forward((embeddings).forward(input_ids, ), attention_mask, )
          ~~~~~~~~~~~~~~~~~~~~ <--- HERE
    return _3
class Embeddings(Module):
  File "code/__torch__/transformers/models/distilbert/modeling_distilbert.py", line 88, in forward
    layer5 = self.layer
    _0 = getattr(layer5, "0")
    _9 = (_0).forward(activation, argument_1, attention_mask, )
          ~~~~~~~~~~~ <--- HERE
    _10 = (_1).forward(activation, _9, attention_mask, )
    _11 = (_2).forward(activation, _10, attention_mask, )
  File "code/__torch__/transformers/models/distilbert/modeling_distilbert.py", line 111, in forward
    sa_layer_norm = self.sa_layer_norm
    attention = self.attention
    _14 = (attention).forward(argument_2, attention_mask, )
           ~~~~~~~~~~~~~~~~~~ <--- HERE
    input = torch.add(_14, argument_2)
    _15 = (sa_layer_norm).forward(input, )
  File "code/__torch__/transformers/models/distilbert/modeling_distilbert.py", line 143, in forward
    k_length = ops.prim.NumToTensor(torch.size(argument_1, 1))
    _22 = int(k_length)
    _23 = torch.view((q_lin).forward(argument_1, ), [_21, -1, 12, 64])
                      ~~~~~~~~~~~~~~ <--- HERE
    q = torch.transpose(_23, 1, 2)
    _24 = torch.view((k_lin).forward(argument_1, ), [_20, -1, 12, 64])
  File "code/__torch__/torch/nn/modules/linear.py", line 12, in forward
    bias = self.bias
    weight = self.weight
    x = torch.linear(argument_1, weight, bias)
        ~~~~~~~~~~~~ <--- HERE
    return x

Traceback of TorchScript, original code (most recent call last):
/usr/local/lib/python3.7/dist-packages/torch/nn/functional.py(1848): linear
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/linear.py(103): forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1090): _slow_forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1102): _call_impl
/usr/local/lib/python3.7/dist-packages/transformers/models/distilbert/modeling_distilbert.py(200): forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1090): _slow_forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1102): _call_impl
/usr/local/lib/python3.7/dist-packages/transformers/models/distilbert/modeling_distilbert.py(277): forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1090): _slow_forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1102): _call_impl
/usr/local/lib/python3.7/dist-packages/transformers/models/distilbert/modeling_distilbert.py(328): forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1090): _slow_forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1102): _call_impl
/usr/local/lib/python3.7/dist-packages/transformers/models/distilbert/modeling_distilbert.py(555): forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1090): _slow_forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1102): _call_impl
/usr/local/lib/python3.7/dist-packages/transformers/models/distilbert/modeling_distilbert.py(734): forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1090): _slow_forward
/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py(1102): _call_impl
/usr/local/lib/python3.7/dist-packages/torch/jit/_trace.py(965): trace_module
/usr/local/lib/python3.7/dist-packages/torch/jit/_trace.py(750): trace
<ipython-input-2-ad335247d066>(14): <module>
/usr/local/lib/python3.7/dist-packages/IPython/core/interactiveshell.py(2882): run_code
/usr/local/lib/python3.7/dist-packages/IPython/core/interactiveshell.py(2822): run_ast_nodes
/usr/local/lib/python3.7/dist-packages/IPython/core/interactiveshell.py(2718): run_cell
/usr/local/lib/python3.7/dist-packages/ipykernel/zmqshell.py(537): run_cell
/usr/local/lib/python3.7/dist-packages/ipykernel/ipkernel.py(208): do_execute
/usr/local/lib/python3.7/dist-packages/ipykernel/kernelbase.py(399): execute_request
/usr/local/lib/python3.7/dist-packages/ipykernel/kernelbase.py(233): dispatch_shell
/usr/local/lib/python3.7/dist-packages/ipykernel/kernelbase.py(283): dispatcher
/usr/local/lib/python3.7/dist-packages/tornado/stack_context.py(300): null_wrapper
/usr/local/lib/python3.7/dist-packages/zmq/eventloop/zmqstream.py(431): _run_callback
/usr/local/lib/python3.7/dist-packages/zmq/eventloop/zmqstream.py(481): _handle_recv
/usr/local/lib/python3.7/dist-packages/zmq/eventloop/zmqstream.py(452): _handle_events
/usr/local/lib/python3.7/dist-packages/tornado/stack_context.py(300): null_wrapper
/usr/local/lib/python3.7/dist-packages/tornado/platform/asyncio.py(122): _handle_events
/usr/lib/python3.7/asyncio/events.py(88): _run
/usr/lib/python3.7/asyncio/base_events.py(1786): _run_once
/usr/lib/python3.7/asyncio/base_events.py(541): run_forever
/usr/local/lib/python3.7/dist-packages/tornado/platform/asyncio.py(132): start
/usr/local/lib/python3.7/dist-packages/ipykernel/kernelapp.py(499): start
/usr/local/lib/python3.7/dist-packages/traitlets/config/application.py(846): launch_instance
/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py(16): <module>
/usr/lib/python3.7/runpy.py(85): _run_code
/usr/lib/python3.7/runpy.py(193): _run_module_as_main
RuntimeError: CUDA error: CUBLAS_STATUS_NOT_INITIALIZED when calling `cublasCreate(handle)`
```

**Comment by deadeyegoodwin** [comment]
@youxingtian Are you able to run this model outside of triton and model analyzer? You may need to send real inputs to the model for it to operate correctly: https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md#real-input-data

**Comment by tanmayv25** [comment]
> RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:7 and cuda:0! (when checking argument for argument index in method wrapper__index_select)

@youxingtian  Which version are you using? There was a fix that went in 22.02 container and might be related.
https://github.com/triton-inference-server/pytorch_backend/pull/49
